modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java

```diff
@@ -217,28 +217,15 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
 				_language.get(_httpServletRequest, "review-change"), "get",
 				"get", null));
 
-		if (_ctCollection.getStatus() == WorkflowConstants.STATUS_EXPIRED) {
-			fdsActionDropdownItems.add(
-				new FDSActionDropdownItem(
-					PortletURLBuilder.createRenderURL(
-						_renderResponse
-					).setMVCRenderCommandName(
-						"/change_tracking/view_move_changes"
-					).setRedirect(
-						_themeDisplay.getURLCurrent()
-					).setParameter(
-						"ctCollectionId", "{ctCollectionId}"
-					).setParameter(
-						"modelClassNameId", "{modelClassNameId}"
-					).setParameter(
-						"modelClassPK", "{modelClassPK}"
-					).buildString(),
-					"move-folder", "move-changes",
-					_language.get(_httpServletRequest, "move-changes"), "post",
-					"move-changes", null));
-		}
+		boolean statusDraft = false;
 
 		if (_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) {
+			statusDraft = true;
+		}
```

Comment:
Tried to simplify by just adding the content directly to the boolean 
`boolean statusDraft =  _ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT `, but SourceFormat didn't allow it and changed to code to this current version

Reply:
Can you try to do it without creating a variable? Simply do 
```
if ((_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) ||
	(_ctCollection.getStatus() == WorkflowConstants.STATUS_EXPIRED))
```